demo/package-lock.json

```diff
@@ -44,11 +44,12 @@
       }
     },
     "node_modules/@babel/code-frame": {
```

Comment:
I don't think we need to change the lock file at all.

Reply:
It's a minimal npm audit fix. mainly for the vite and babel security issue.

Reply:
I think we should only change what's necessary. Keeps the diff sane over the long-term.